docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1292,7 +1292,7 @@ function clearMeetingsResolutionCheckInterval() {
 // Meeting Streams --------------------------------------------------
 
 function addMediaOptions(elementId) {
-  const mediaOptions = ['360p', '480p', '720p', '1080p', 'LOW', 'MEDIUM', 'HIGH'];
+  const mediaOptions = ['360p', '480p', '720p', '1080p'];
```

Comment:
Removed 'LOW', 'MEDIUM', 'HIGH' from the sample's selection.

Reply:
Remote will have LOW', 'MEDIUM', 'HIGH only

Reply:
Are we keeping 360p?

Reply:
Yes @rajeshtezu90,